Cargo.toml

```diff
@@ -1,14 +1,22 @@
 [package]
 name = "embedded-nal"
 version = "0.1.0"
-authors = ["Jonathan 'theJPster' Pallant <github@thejpster.org.uk>"]
+authors = [
+    "Jonathan 'theJPster' Pallant <github@thejpster.org.uk>",
+    "Mathias Koch <mk@blackbird.online>",
+    "Diego Barrios Romero <eldruin@gmail.com>",
+]
 edition = "2018"
 description = "A Network Abstraction Layer (NAL) for Embedded Systems"
 license = "MIT or Apache-2.0"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+repository = "https://github.com/rust-embedded-community/embedded-nal"
+homepage = "https://github.com/rust-embedded-community/embedded-nal"
+documentation = "https://docs.rs/embedded-nal"
+readme = "README.md"
+keywords = ["network"]
+categories = ["embedded", "hardware-support", "no-std", "network-programming"]
 
 [dependencies]
-nb = "0.1"
-no-std-net = "0.3.0"
+nb = "1"
```

Comment:
One thing I noticed during development is that the traits require `nb::Result`, but `nb::Result` (or `nb`) is not re-exported by the crate, which can make a weird dependency situation. Can we add a `pub use nb;` to `lib.rs`?

Reply:
Sure thing. I added it now. thanks!